resources/migrations/000_migrations.yaml

```
@@ -16064,6 +16064,27 @@ databaseChangeLog:
             newDataType: VARCHAR(32)
       rollback: # not necessary
 
+  - changeSet:
+      id: v48.00-054
```

`v48.00-054` was claimed already (by Bryan) so the next available ID is 055. I just claimed it in #migrations
```suggestion
      id: v48.00-055
```